code/modules/power/supermatter/supermatter.dm

```diff
@@ -405,7 +405,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 
 	if(!removed || !removed.total_moles() || isspaceturf(T)) //we're in space or there is no gas to process
 		if(takes_damage)
-			damage += max((power / 1000) * DAMAGE_INCREASE_MULTIPLIER, 0.1) // always does at least some damage
+			damage += max((power / 400) * DAMAGE_INCREASE_MULTIPLIER, 0.1) // always does at least some damage
```

Comment:
This doesn't really matter much as spacing the supermatter usually causes power to decrease very fast to irrelevant levels.